tests/server/helpers/index.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+  isDocData: function(response) {
+    var defined = false;
+    for (var key in response) {
+      var doc = response[key];
+
+      if (doc.title) {
+        defined = true;
+      } else if (doc.content) {
+        defined = true;
+      } else if (doc.dateCreated) {
+        defined = true;
+      } else if (doc.lastModified) {
+        defined = true;
+      }
+    }
+    return defined;
+  },
+
+  isUserData: function(response) {
+    var defined = false;
+    for (var key in response) {
```

Comment:
The body of a for in should be wrapped in an if statement to filter unwanted properties from the prototype.